.github/styles/Vocab/HMSVocab/accept.txt

```diff
@@ -113,4 +113,6 @@ hls.started.success
 hls_vod_recording_path
 hls_vod_recording_presigned_url
 session_duration
-px
\ No newline at end of file
+px
+room_id
```

Comment:
let's not add this here. use backticks or "room ID"

Reply:
fixed it.